### DISCUSSION TITLE
Porting Mix 3 5 G

### DISCUSSION THREAD

**Discussion by Dkorg2022** [community/community#49772] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Hi, I have a mix 3 5G I wanted to Install windows 11 on it but need boot.img Adromeda. can anyone help me on this?

**Comment by otter-computer** [comment]
Welcome to the GitHub Community, @Dkorg2022, we're happy you're here! You are more likely to get a useful response if you are posting your question(s) in the applicable category and are explicit about what your project entails--giving a few more details might help someone give you a nudge in the right direction. I've gone ahead and moved it for you. Good luck!
